modules/reference/pages/command-line-tools.adoc

```diff
@@ -761,6 +761,29 @@ unset URLGRABBER_DEBUG
 
 
 
+=== Add Custom Extra HTTP Headers
+
+It is possible to add custom HTTP headers to the requests made by [command]``spacewalk-repo-sync`` at the time of repository and package synchronization.
+
+The custom HTTP headers are defined in the [path]``/etc/rhn/spacewalk-repo-sync/extra_headers.conf`` configuration file. 
+The headers can be defined by repository name, or channel label.
+You can also define global headers by putting them in the ``main`` section:
+
+----
+[testchannel]
+X-MY-HEADER-1=VALUE
+X-MY-HEADER-2=VALUE
+
+[mychannel]
+X-MY-HEADER-3=VALUE
+X-MY-HEADER-4=VALUE
+
+[main]
+X-GLOBAL-HEADER=VALUE
```

Comment:
For consistency with the other examples and to make it clear more than one global header is possible, I'd suggest "x-myglobal..." instead of "x-global..."

```suggestion
X-MYGLOBAL-HEADER=VALUE
```